pimd/pim_iface.c

```diff
@@ -1546,9 +1546,9 @@ void pim_if_create_pimreg(struct pim_instance *pim)
 
 	if (!pim->regiface) {
 		if (pim->vrf->vrf_id == VRF_DEFAULT)
-			strlcpy(pimreg_name, "pimreg", sizeof(pimreg_name));
+			strlcpy(pimreg_name, "PIMREG", sizeof(pimreg_name));
```

Comment:
Interface names all-capital? And is this going to be used in JSON outputs as well as the key for interface names?

Reply:
This was a typo-mistake. fixed now.